src/utils/__tests__/__snapshots__/publicAPI.test.js.snap

```diff
@@ -293,7 +293,7 @@ Map {
           "onChangeSort": [Function],
           "onColumnResize": [Function],
           "onColumnSelectionConfig": [Function],
-          "onEmptyStateAction": [Function],
+          "onEmptyStateAction": null,
```

Comment:
I think the default we've established elsewhere is a noop `()=>{}` instead of `null`

Reply:
This will probably bork your conditional logic for the callback, but I'm not sure we should rely on a null check for that anyway.

`()=>{}` returns `undefined`, maybe that's a more appropriate default? but I'm unsure how that would jive with the proptypes declaration for this.

Thoughts?

Reply:
Unfortunately I'd have to call the function to determine whether to render or not and then it would actually fire the function.  I think the noop on functions by defaults is a kind of strange pattern that keeps us from being able to conditionally callbacks or make render decisions based on functional props, which means that our prop model becomes more complicated than it needs to in several cases.